src/MultiplayerMod/Multiplayer/CoreOperations/Events/ConnectionToHostHasBeenLost.cs

```diff
@@ -0,0 +1,5 @@
+using MultiplayerMod.Core.Events;
+
+namespace MultiplayerMod.Multiplayer.CoreOperations.Events;
+
+public record ConnectionToHostHasBeenLost : IDispatchableEvent;
```

Comment:
Maybe something like `MultiplayerConnectionLostEvent` or at least without `has been`.

Reply:
Renamed to ConnectionLostEvent.

In general I'm trying to avoid usage of multiplayer word since our project name and various network related layers its loosing any meaning.